demo/src/main/java/com/midtrans/demo/DemoSavedCardsActivity.java

```diff
@@ -125,19 +123,19 @@ private void checkSavedCards() {
         }
     }
 
-    private void fetchSavedCards() {
-
-        try {
-            SavedCards savedCards = LocalDataHandler.readObject(SAVED_CARDS_TYPE, SavedCards.class);
-            List<SavedCard> loadedSaveCards = savedCards.savedCards;
-            if (loadedSaveCards != null && !loadedSaveCards.isEmpty()) {
-                this.savedCards.clear();
-                this.savedCards.addAll(loadedSaveCards);
-            }
-        } catch (Exception e) {
-            e.printStackTrace();
-        }
-    }
+//    private void fetchSavedCards() {
+//
+//        try {
+//            SavedCards savedCards = LocalDataHandler.readObject(SAVED_CARDS_TYPE, SavedCards.class);
+//            List<SavedCard> loadedSaveCards = savedCards.savedCards;
+//            if (loadedSaveCards != null && !loadedSaveCards.isEmpty()) {
+//                this.savedCards.clear();
+//                this.savedCards.addAll(loadedSaveCards);
+//            }
+//        } catch (Exception e) {
+//            e.printStackTrace();
+//        }
+//    }
```

Comment:
Same here. We're using git, can always revert

Reply:
we still need DemoSaveCards??

Reply:
can we make demo apps easier like demo snap web, there is no demo for saving cards and the user detail can't edit

Reply:
the purpose of demo app on our side is to help the QA since we don't publish it anymore. However for saved card we can use the cards from the response, No?

Reply:
yes for the saved card we can use the response when we doing the transaction. previously `DemoSavedCardActivity` only doing save the card to `LocalData`, I don't understand what the purpose, in real life this feature impossible to happen because Snap will be doing save card when the transaction happens.

Reply:
Let's remove that part then since it wont be part of the functionality @askur87

Reply:
86

Reply:
@femmerling please re-check I was removed DemoSavedCardsActivity.java